src/Middleware/WebSockets/test/UnitTests/WebSocketCompressionMiddlewareTests.cs

```diff
@@ -0,0 +1,306 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.WebSockets;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Testing;
+using Microsoft.Net.Http.Headers;
+using Xunit;
+
+namespace Microsoft.AspNetCore.WebSockets.Test
+{
+    public class WebSocketCompressionMiddlewareTests : LoggedTest
```

Comment:
Have you enabled the autobahn tests yet?

Reply:
I ran it earlier, but wanting to get everything cleaned up before trying again

Reply:
All Autobahn tests passed